packages/quantic/force-app/main/default/lwc/quanticResultLink/quanticResultLink.js

```diff
@@ -1,5 +1,5 @@
+import navigateToRecord from '@salesforce/label/c.quantic_NavigateToRecord';
 import opensInBrowserTab from '@salesforce/label/c.quantic_OpensInBrowserTab';
-import opensInSalesforceSubTab from '@salesforce/label/c.quantic_OpensInSalesforceSubTab';
```

Comment:
we should also remove the `quantic_OpensInSalesforceSubTab` label and its translation since they will now not be used anywhere in the package

Reply:
Removing a label that was previously part of the package could create a breaking change. This I wouldn't remove it now. it would be safe to remove in the next v3.